showpass/Showpass-E2E-tests
# Create the 'cypress-cloud.yml' workflow

**Description:**
We need an ability to record test executions in the Cypress Cloud platform.
Created a workflow to achieve this.

**How to test:**
Run the  'cypress-cloud.yml' workflow and ensure that test runs get recorded in the cloud.

**Test execution results:**
